orc8r/cloud/go/services/tenants/obsidian/handlers/handlers.go

```diff
@@ -103,11 +114,9 @@ func GetTenantHandler(c echo.Context) error {
 		return terr
 	}
 	tenantInfo, err := tenants.GetTenant(c.Request().Context(), tenantID)
-	switch {
-	case err == errors.ErrNotFound:
-		return echo.NewHTTPError(http.StatusNotFound, fmt.Errorf("Tenant %d does not exist", tenantID))
-	case err != nil:
-		return echo.NewHTTPError(http.StatusInternalServerError, err)
+	err = errorHandlingForGet(err, fmt.Errorf("Tenant %d does not exist", tenantID), err)
```

Comment:
Same idea here -- call this `mapErr` (or `mapGetErr` if necessary), and only perform the wrap as part of the return, so e.g. `return mapErr(err)`

Reply:
this is only for get though, (otherwise would need more paramters), can I call it `mapErrForGet`?

Reply:
I like `mapGetErr` -- conciseness/skim-ability is a huge pro for Go code. But `mapErrForGet` is also fine, up to you 👍

Reply:
But the method functionality could be confused for that "the method gets err"?

Reply:
Up to you 🙂